compiler/testData/codegen/box/callableReference/bound/equals/propertyAccessors.kt

```diff
@@ -29,9 +32,13 @@ fun box() =
             cProp.getter == C_propReflect.getter -> "cProp.getter == C_propReflect.getter"
             C_propReflect.getter == cProp.getter -> "C_propReflect.getter == cProp.getter"
 
-            // TODO https://youtrack.jetbrains.com/issue/KT-13490
-            // cProp.getter != C()::prop.getter -> "cProp.getter != C()::prop.getter"
-            // cProp.setter != C()::prop.setter -> "cProp.setter != C()::prop.setter"
+            // https://youtrack.jetbrains.com/issue/KT-13490
```

Comment:
Now that the issue is fixed, this comment is no longer needed, so please remove it